cpp/example_code/elasticfilesystem/create_file_system_with_modeled_exceptions.cpp

```diff
@@ -0,0 +1,120 @@
+
```

Comment:
Missing unit tests?

Reply:
Could you give me some examples about how to write unit tests for C++ examples?